tests/page-objects/user-settings/user-settings.wdio.page.js

```diff
@@ -0,0 +1,22 @@
+const submiButton = () => $('.btn.submit.btn-primary');
+
+const openEditSettings = async () => {
+  const links = await $('.content .configuration');
+  await links.waitForDisplayed();
+  await (await links.$$('.btn-link'))[1].click();
+  //modals have an animation and the click might land somewhere else
+  await browser.pause(500);
+};
+
+const selectLanguage = async (code) => {
+  const languageDropDown = await $('#language');
+  await browser.waitUntil(async () => await languageDropDown.getValue() === 'en');
+  await languageDropDown.selectByAttribute('value', code);
```

Comment:
To prevent stale element errors, I think this has to be:
```suggestion
const languageDropDown = () => $('#language');
await browser.waitUntil(async () => await (await languageDropDown()).getValue() === 'en');
```